pocs/tests/test_pocs.py

```diff
@@ -51,18 +52,23 @@ def scheduler(config):
 
 
 @pytest.fixture(scope='function')
-def dome(config):
-    return create_dome_from_config(config)
+def dome(config_with_simulated_dome):
+    return create_dome_from_config(config_with_simulated_dome)
 
 
 @pytest.fixture(scope='function')
-def observatory(config, db_type, cameras, scheduler, dome):
+def mount(config_with_simulated_mount):
+    return create_mount_from_config(config_with_simulated_mount)
+
+
+@pytest.fixture(scope='function')
+def observatory(config_with_simulated_mount, db_type, cameras, scheduler, dome, mount):
     observatory = Observatory(
-        config=config,
+        config=config_with_simulated_mount,
```

Comment:
Since you are passing in the created mount simulator do you still need this config?